egs/wsj/s5/RESULTS

```diff
@@ -277,4 +277,16 @@ for x in exp/nnet2_online/nnet_ms_a_online/decode_*; do grep WER $x/wer_* | util
 %WER 6.68 [ 377 / 5643, 102 ins, 13 del, 262 sub ] exp/nnet2_online/nnet_ms_a_online/decode_tgpr_eval92_utt/wer_10
 %WER 6.56 [ 370 / 5643, 100 ins, 12 del, 258 sub ] exp/nnet2_online/nnet_ms_a_online/decode_tgpr_eval92_utt_offline/wer_10
 
-
+# LSTM results:
```

Comment:
These results are not comparable to the nnet2_online(old tdnn) results above as we are using model-discarding in the new TDNN recipe. Could you run the TDNN recipe with the new steps/nnet3/train_tdnn.sh and add the results here.